src/core/network.js

```diff
@@ -295,13 +295,331 @@ var NetworkManager = (function NetworkManagerClosure() {
 //#if !(FIREFOX || MOZCENTRAL)
 (function (root, factory) {
   if (typeof define === 'function' && define.amd) {
-    define('pdfjs/core/network', ['exports'], factory);
+    define('pdfjs/core/network', ['exports', 'pdfjs/shared/util',
+      'pdfjs/core/worker'], factory);
   } else if (typeof exports !== 'undefined') {
-    factory(exports);
+    factory(exports, require('../shared/util.js'), require('./worker.js'));
   } else {
-    factory((root.pdfjsCoreNetwork = {}));
+    factory((root.pdfjsCoreNetwork = {}), root.pdfjsSharedUtil,
+      root.pdfjsCoreWorker);
   }
-}(this, function (exports) {
+}(this, function (exports, sharedUtil, coreWorker) {
+
+  var assert = sharedUtil.assert;
+  var createPromiseCapability = sharedUtil.createPromiseCapability;
+  var isInt = sharedUtil.isInt;
+  var MissingPDFException = sharedUtil.MissingPDFException;
+  var UnexpectedResponseException = sharedUtil.UnexpectedResponseException;
+
+  function PDFNetworkStream(options) {
+    this._options = options;
+    var source = options.source;
+    this._manager = new NetworkManager(source.url, {
+      httpHeaders: source.httpHeaders,
+      withCredentials: source.withCredentials
+    });
+    this._rangeChunkSize = source.rangeChunkSize;
+    this._fullRequestReader = null;
+    this._rangeRequestReaders = [];
+  }
+
+  PDFNetworkStream.prototype = {
+    _onRangeRequestReaderClosed: function (reader) {
```

Comment:
All of these function members are anonymous. According to [our style guide](https://github.com/mozilla/pdf.js/wiki/Style-Guide#classes), it should be something like `PDFNetworkStream__onRangeRequestReaderClosed`.